client/src/common/components/Home.tsx

```diff
@@ -1,8 +1,149 @@
 import React from 'react';
+import { Link } from 'react-router-dom';
+
+import {
+	Accordion,
+	AccordionItem,
+	AccordionItemHeading,
+	AccordionItemButton,
+	AccordionItemPanel,
+} from 'react-accessible-accordion';
+import 'react-accessible-accordion/dist/fancy-example.css';
 
 const Home = () => {
 	// test, added style here just to see the classes work
-	return <div　className="bg-primary_bg_color w-screen h-96 mt-headerHeight text-primary_title_color">This is Home</div>;
+	return (
+		<div className="bg-primary_bg_color text-primary_title_color bg-gradient">
+			<div className="p-10 md:p-20">
+				<div className="md:float-right w:20 md:w-2/4 ">
+					<img src="./assets/shared_workspace.svg" />
+				</div>
+
+				<div className="text-3xl font-black md:text-7xl mt-5 text-center md:text-justify">
+					Senpai Kohai
+				</div>
+				<div className="mt-5 font-extrabold text-base w-auto md:text-2xl text-center md:text-justify">
+					Find your guide, Support youngs
+				</div>
+				<div className="underline cursor-pointer mt-5 text-sm md:text-xl text-center md:text-justify">
+					<span className="">What are Senpai and Kohai?</span>
+				</div>
+				<div className="flex justify-center items-center md:justify-start">
+					<div
+						className="cursor-pointer 
+								mt-5 
+								border-2 
+								w-24 
+								text-primary_bg_color 
+								bg-primary_title_color 
+								rounded-full 
+								py-3 
+								font-black 
+								hover:bg-secondary_bg_color 
+								text-center
+								text-sm
+								justify-center
+								md:text-base
+								"
+					>
+						<Link to="/signup">Join Now</Link>
+					</div>
+				</div>
+			</div>
+			<div className="text-center p-10 md:p-20 bg-BlobL bg-no-repeat bg-contain">
```

Comment:
@Dannyboy222 
Not using the blob image as a `background-image`.  Suggest that you set the blob image as an element with `img` and adjust the position with `absolute`, `top` or `left` attributes.

Probably the entire container will need to have `overflow-x: hidden` to chop off some part of the blob.